data/campaigns/Under_the_Burning_Suns/units/quenoth/Champion.cfg

```diff
@@ -24,6 +24,9 @@
     description= _ "Standing at the forefront of most Quenoth spear lines, Champions are those who have mastered the use of the glaive to near perfection. Perhaps only slightly lacking in finesse in strategy, these elves more than compensate with raw power and can force their way through all but the most secure of defenses."
     die_sound={SOUND_LIST:ELF_HIT}
     attacks=5
+    # Note: Key character Garak, based on the Champion, has an event on turn 2 of S3 which converts his melee attacks to fire damage.
+    # If changing the attack definitions for the Champion, be sure to review and update the corresponding event in S3.
+    # Reference: GitHub issue #7774
```

Comment:
Could this be avoided by instead using
```
[not]
    type=arcane
[/not]
```
?

Reply:
It would appear so.